apps/dav/lib/CalDAV/Export/ExportService.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+namespace OCA\DAV\CalDAV\Export;
+
+use Generator;
+use OCP\Calendar\CalendarExportOptions;
+use OCP\Calendar\ICalendarExport;
+use OCP\IConfig;
+use Sabre\VObject\Component;
+use Sabre\VObject\Writer;
+
+/**
+ * Calendar Export Service
+ */
+class ExportService {
+	
+	public const FORMATS = ['ical', 'jcal', 'xcal'];
+	public string $systemVersion = '0.0';
+
+	public function __construct(IConfig $config) {
```

Comment:
```suggestion
	public function __construct(ServerVersion $serverVersion) {
```

Let's use the right api to fetch the server version